sklearn/multiclass.py

```diff
@@ -160,8 +160,7 @@ class OneVsRestClassifier(MultiOutputMixin, ClassifierMixin,
         :term:`decision_function` or :term:`predict_proba`.
 
     n_jobs : int or None, optional (default=None)
-        The number of jobs to use for the computation, where each job computes
-        a classifier in parallel.
+        The number of jobs to use for the computation: the n_classes one-vs-rest problems are computed in parallel.
```

Comment:
This needs to rewrap to make sure we are under 80 characters per line.

Reply:
Sure @thomasjpfan. I think I have updated it already. Can you please verify the new commit.